benchmarks/benchmarks/linalg.py

```diff
@@ -56,6 +56,12 @@ def time_solve(self, size, contig, module):
         else:
             sl.solve(self.a, self.b)
 
+    def time_solve_triangular(self, size, contig, module):
```

Comment:
Consider commenting that this treats `self.a` as if it were a lower-triangular matrix by ignoring the above-diagonal entries.